source/simulator/assembly.cc

```diff
@@ -804,6 +913,14 @@ namespace aspect
                                                parameters.composition_degree)
                                               +
                                               (parameters.stokes_velocity_degree+1)/2),
+                                  //TODO: give this a 1-point QGauss to save space, as this is never going to be used here? (discontinuous temperature field doesn't require stabilization)
+                                  QGauss<dim-1>((advection_field.is_temperature()
```

Comment:
Fixed to only assemble face terms if advection_field is discontinuous (and never in `get_artificial_viscosity`) in e4844ff.